ffi-libarchive.gemspec

```diff
@@ -15,5 +15,7 @@ Gem::Specification.new do |s|
   s.files = %w{ LICENSE } + Dir.glob("lib/**/*", File::FNM_DOTMATCH).reject { |f| File.directory?(f) }
   s.require_paths = %w{lib}
   s.required_ruby_version = ">= 3.0"
-  s.add_dependency "ffi", "~> 1.0"
+  s.add_dependency "ffi", "~> 1.17"
+
+  s.add_development_dependency "cookstyle", "~> 8.1"
```

Comment:
Just a future heads up the reason these are in the gemfile is so they can get excluded in the omnibus builds via groups so you don't ship a bunch of gems in the packages you don't need.